lib/jsdom/living/interfaces.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable global-require */
+"use strict";
+
+const DOMException = require("domexception");
+const { URL, URLSearchParams } = require("whatwg-url");
+const XMLSerializer = require("w3c-xmlserializer/lib/XMLSerializer").interface;
+
+const registerElements = require("./register-elements");
+const style = require("../level2/style");
+const xpath = require("../level3/xpath");
+const nodeFilter = require("./node-filter");
+
+const GENERATED_INTERFACES = [
```

Comment:
No need to SHOUTY_CASE this.

Reply:
But isn’t it a common convention that constants are in `UPPER_SNAKE_CASE`?